README.md

```diff
@@ -48,7 +68,7 @@ Before submitting your pull request, you'll want to verify the changes in two wa
 All the tests for GNU APL exercises can be run from the top level of the repo with
 
 ```
-# add this command
+bin/run-all-tests
 ```
 
 For the Exercism-specific linting, please see [the documentation](https://github.com/exercism/docs/blob/master/language-tracks/configuration/linting.md).
```

Comment:
What's the deal with this? The link is broken, and I'm not sure what this is about.

Reply:
The docs are a bit of a mess at the moment :/ it's described here: https://github.com/exercism/docs/blob/a8ea5db92e2a2d2839e66ec10c7687b3b7db002a/language-tracks/configuration/configlet.md

Reply:
OK. I'll look there.